codejam-canvas/assets/js/index.js

```diff
@@ -0,0 +1,40 @@
+draw(frame_4x4, false); //default picture
+
+function draw(frame, isImage){
+    var canvas=document.getElementById('canvas');
```

Comment:
Please also leave empty spaces between operators/variables/function calls everywhere. It increase the readability.
`var canvas = document.getElementById('canvas');`